Link to Umbraco backoffice site from CMS (#5034)
umbraco/UmbracoDocs: 11/umbraco-cms/extending/backoffice-ui-api-documentation.md

A library of API Reference documentation is auto-generated from the comments within the Umbraco Source Code.

{% hint style="info" %}

> A new backoffice for Umbraco CMS is currently being worked on. It is planned for release with Umbraco 13 i December of 2023.

Reviewer: ```suggestion
A new backoffice for Umbraco CMS is currently being worked upon. It is planned to release with Umbraco version 13 in December 2023.
```

We have gathered all information and resources about the project in one place, giving you a full overview of what we're working on. You will also be able to find information about how to get involved in the project.

> You can find it all on [docs.umbraco.com/umbraco-backoffice](https://docs.umbraco.com/umbraco-backoffice).

Reviewer: ```suggestion
You can find it in the [The new Umbraco Backoffice](https://docs.umbraco.com/umbraco-backoffice) article.
```
